python/CHANGES.rst

```diff
@@ -1,6 +1,12 @@
 Changelog for tchannel.py
 =========================
 
+0.9.2 (unreleased)
```

Comment:
If you made a breaking change it should be a bump. Personally I don't think this is breaking but you would know better than me.

If it's not a breaking change (just a patch), it'll go under 0.9.2 and diffs will pile up there until we cut a new release.